package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@salesforce/core",
-  "version": "2.9.3",
+  "version": "2.9.4",
```

Comment:
We use semantic release here so this automatically happens, along with the CHANGELOG.md.